quickstart-short.md

```diff
@@ -0,0 +1,273 @@
+- working with faucets and funding wallets
+
+**Developers** interested in development with Livepeer, should also have a working knowledge of:
+
+- Go language
+- Git
+- Building from source, binaries, and Docker    
+
+## Technical Requirements
+
+Livepeer supports the following to install and run `livepeer` on the network:  
+
+**Operating Systems** 
+
+Linux, Darwin (macOS), and Windows
+
+**Hardware** 
+
+Livepeer supports [NVIDIA **GPUs**](/video-miners/reference/gpu-support) for encoding/decoding. 
+
+A [concurrency check](/video-miners/reference/concurrency-check#hardware-functionality-and-constraints) is recommended to assess concurrent streams capabilities.
+
+**Network** 
+
+- **Bandwidth** should provide a recommended minimum 1G upload/download. [Test your available bandwith](/video-miners/reference/bandwidth).
+- **Static IP/Hostname** to receive video streams 
+(Dynamic IP will suffice to get started, but should be static for ongoing use)
+- **Firewall and Routing** Ability to open ports and modify port forwarding.
+
+> **NOTE** For further detail, refer to [Common Questions](/video-miners/how-to-guides/troubleshooting#common-questions).
+
+## Installation Workflow
+
+**Current Releases** for supported operating systems are found on our Github repository [releases page](https://github.com/livepeer/go-livepeer/releases).
+
+Depending on your workflow, you can install Livepeer:
+> * [Using a binary release](/installation/install-livepeer/binary-release)
+> * [Build from a Docker image](/installation/install-livepeer/docker)
+> * [Perform a Developer Install](/installation/install-livepeer/installing-for-development)
+
+## Start Video Mining
+
+You will be setting up a standalone Orchestrator with a transcoder so that you will be able to manage it, get it listed on the Orchestrator list and view it on the Livepeer Explorer.
+
+### Considerations 
+
+To begin with, we will be running on the Arbitrum Rinkeby testnet. You will need to acquire token to be able to get started.
+
+- [Connect a Wallet](/video-miners/how-to-guides/l2-migration#migrating-to-arbitrum)
+
+    - Arbitrum Rinkeby testnet, Rinkeby ETH is made available free of charge.
+
+    - On Arbitrum (LP2 mainnet) you will need to purchase ETH to cover gas fees and LPT to stake and activate on mainnet. (See ETH and LPT)
+
+- Decide how you want to participate on the network. ***See*** [About Video Mining](/video-miners/core-concepts/roles-and-responsibilities#types-of-video-miners).
+
+### Choose your role:
+
+- [Orchestrator](/video-miners/getting-started/choosing-a-role#orchestrator)
+- [Transcoder](/video-miners/getting-started/choosing-a-role#transcoder)
+
+### Download and [Install](/installation/livepeer-quickstart-shortversion#installation-workflow) `livepeer`
+
+**For Example** Livepeer Binary should be saved at a location of your choice and includes:
+
+    - `livepeer.exe`
+    - `livepeer_bench.exe`
+    - `livepeer_cli.exe`
+    - `livepeer_router.exe`

```

Comment:
should we just sent people to the install link? i got a little confused here as well